src/pathutils.rs

```diff
@@ -0,0 +1,25 @@
+use crate::shell::Shell;
+use std::process::Command;
+
+#[cfg(windows)]
+pub fn format_path(path: &str, shell: &str) -> String {
+    match shell {
+        "cmd" | "pwsh" => String::from(path),
+        _ => {
+            let output = Command::new("cygpath")
```

Comment:
Will this solve WSL issues too like #758? Is `cygpath` a part of WSL?

Reply:
No, `cygpath` comes from Cygwin. It's included in Cygwin, MSYS2, and Git Bash (which are the most common ways of using bash/zsh/fish on Windows natively. It doesn't exist on WSL, but from a bit of searching I found that WSL does include a `wslpath` command that functions in the same way (but adding a `/mnt` in front to make it WSL compatible) which we could probably use if we can detect if a Windows executable is being used in WSL.